common/src/main/java/com/wynntils/features/user/GuildStarsToRankFeature.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.user;
+
+import com.wynntils.core.features.UserFeature;
+import com.wynntils.wynn.event.ChatMessageReceivedEvent;
+import com.wynntils.wynn.objects.GuildRank;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import net.minecraft.network.chat.MutableComponent;
+import net.minecraft.network.chat.TextComponent;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+
+public class GuildStarsToRankFeature extends UserFeature {
```

Comment:
IMO this feature is one that should be disabled by default via `@StartDisabled` - it's a subjective change to a core wynncraft feature